auth-web/src/views/AuthHome.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="view-container">
+    <article>
+      <p class="intro-text"><span><h1>Quick and easy access to BC Government and partner services for legal, business, government and professionals.</h1></span></p>
+
+        <v-btn class="sign-in-btn" @click="login" color="primary" large to="/createaccount">
+          <span>Create an Account</span>
+        </v-btn>
+        <v-btn class="sign-in-btn" @click="login" color="primary" large>
+          <span>Learn More</span>
+        </v-btn>
+    </article>
+
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+
+export default ({
```

Comment:
use @component ?

Reply:
Removed it completely, it's just a static view